samples/matter/thermostat/README.rst

```diff
@@ -0,0 +1,352 @@
+
+.. include:: /includes/build_and_run.txt
+
+Selecting a build type
+======================
+
+Before you start testing the application, you can select one of the `Matter thermostat build types`_, depending on your building method.
+
+Selecting a build type in |VSC|
+-------------------------------
+
+.. include:: /config_and_build/modifying.rst
+   :start-after: build_types_selection_vsc_start
+   :end-before: build_types_selection_vsc_end
+
+Selecting a build type from command line
+----------------------------------------
+
+.. include:: /config_and_build/modifying.rst
+   :start-after: build_types_selection_cmd_start
+   :end-before: For example, you can replace the
+
+For example, you can replace the *selected_build_type* variable to build the ``release`` firmware for ``nrf52840dk_nrf52840`` by running the following command in the project directory:
+
+.. parsed-literal::
+   :class: highlight
+
+   west build -b nrf52840dk_nrf52840 -d build_nrf52840dk_nrf52840 -- -DCONF_FILE=prj_release.conf
+
+The ``build_nrf52840dk_nrf52840`` parameter specifies the output directory for the build files.
+
+.. note::
+   If the selected board does not support the selected build type, the build is interrupted.
+   For example, if the ``shell`` build type is not supported by the selected board, the following notification appears:
+
+   .. code-block:: console
+
+      File not found: ./ncs/nrf/samples/matter/thermostat/configuration/nrf52840dk_nrf52840/prj_shell.conf
+
+.. _matter_thermostat_testing:
+
+Testing
+=======
+
+After building the sample and programming it to your development kit, you can either test the sample's basic features or use the Matter weather station application to :ref:`test the thermostat with an external sensor <matter_thermostat_sensor_testing>`.
+
+.. _matter_thermostat_sample_basic_features_tests:
+
+Testing basic features
+----------------------
+
+After building the sample and programming it to your development kit, complete the following steps to test its basic features:
+
+1. |connect_kit|
+#. |connect_terminal_ANSI|
+#. Observe that the **LED1** starts flashing (short flash on).
+   This means that the sample has automatically started the Bluetooth LE advertising.
+#. Observe the UART terminal.
+   The sample starts automatically printing the simulated temperature data to the terminal with 30-second intervals.
+#. Press **Button 2** to print the thermostat temperature data to the terminal.
+#. Press **Button 1** for six seconds to initiate the factory reset of the device.
+
+The device reboots after all its settings are erased.
+
+.. _matter_thermostat_sensor_testing:
+
+Testing with external sensor
+----------------------------
+
+After building this sample and the :ref:`Matter weather station <matter_weather_station_app>` application and programming them to the development kits, complete the following steps to test communication between both devices:
+
+1. |connect_kit|
```

Comment:
"Connect the kit to the computer using a USB cable" -> which kit? I guess we should clarify which kit is which (DK vs. Thingy).

Reply:
Thingy is a "prototyping platform", so I guess kit = DK. This means line 248 needs also an edit.